patch.py

```diff
@@ -0,0 +1,53 @@
+# patch.py
+import amy
+
+class Patch:
+    patch_num = 1024
+    def new_patch_num():
+        # just keep a global patch num around for these patches
+        Patch.patch_num = Patch.patch_num + 1
+        return Patch.patch_num-1
+
+    def synth_with_patch_func(func,synth=0):
+        p = func()
+        amy.send(synth=0, patch_number=p)
+        return synth
+        
+    def simple_sine(**kwargs):
+        p = Patch.new_patch_num()
+        amy.send(osc=0, wave=amy.SINE, bp0="10,1,240,0.7,500,0", patch_number=p,**kwargs)
```

Comment:
I think it reads better if you put the `patch_number` as the first arg, as it kinda changes the meaning of everything following